core/templates/dev/head/pages/topic_editor/subtopics_editor/SubtopicsListTabDirective.js

```diff
@@ -73,26 +81,28 @@ oppia.directive('subtopicsListTab', [
                   };
 
                   $scope.updateSubtopicTitle = function(title) {
+                    if (title === subtopic.getTitle()) {
+                      return;
+                    }
+                    if ($scope.subtopicTitles.indexOf(title) !== -1) {
+                      $scope.errorMsg =
+                        'Subtopic with the title already exists';
```

Comment:
A subtopic with this title already exists.

Reply:
Done.